src/Microsoft.DotNet.Interactive.PowerShell.Tests/PowerShellKernelTests.cs

```diff
@@ -240,5 +240,26 @@ public async Task Powershell_customobject_is_formatted_for_outdisplay()
                 e => e.Should().BeOfType<CommandSucceeded>()
             );
         }
+
+        [Fact]
+        public async Task RequestValueInfos_only_returns_user_defined_values()

```

Comment:
Can we explicitly request a non-user defined value?

Reply:
Yes, it can be requested, but it's not reported by default, otherwise the variable explorer is flooded with crap.

Reply:
E.g., PowerShell has a bunch of pre-defined values like `CurrentEncoding`, `True`, `False`, etc.